src/app/random-name/name.ts

```diff
@@ -0,0 +1,3 @@
+export interface Name {
+  value: string;
```

Comment:
Interfejs nadal jest zły:

export type Name = string; - to jest poprawny interfejs

Reply:
Poprawione